data-prepper-plugins/opensearch/src/integrationTest/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSinkIT.java

```diff
@@ -96,6 +100,17 @@ public class OpenSearchSinkIT {
   @Mock
   private PluginFactory pluginFactory;
 
+  @Mock
+  private AcknowledgementSetManager acknowledgementSetManager;
+
+  @BeforeEach
+  public void setup() {
+    acknowledgementSetManager = mock(AcknowledgementSetManager.class);
+    lenient().doAnswer(a -> {
```

Comment:
Since this is a static value, you should be able to have:

```
lenient().when(acknowledgementSetManager).releaseEventReference(any(), any(Boolean.class)).thenReturn(null);
```

Reply:
It doesn't seem to work. Since the function returns void, we can't do `thenReturn` I think.

Reply:
Sorry, I didn't realize it was `void`. I think you don't need these three lines at all in this case. It isn't doing anything from what I can see.